app/jobs/lakeshore/test_fail_job.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+class Lakeshore::TestFailJob < ActiveJob::Base
```

Comment:
Yeah, the testing that goes along with this isn't okay yet, but it's testing a a pretty benign configuration. I do want to spend more time with it and have some sort of test that confirms the new configuration that logs to a different log actually logs to it, but I won't have the time to do that for a few more days, and there's a need for it before then. I could remove these tests since they're not really doing much, until I have them in shape that will support the application.